Cleanup

maansaake opened the pull request.

maansthoernvik commented on backend/templates/index.html:
```diff
@@ -34,7 +34,9 @@
     <app-root></app-root>
-
+    
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```
Vad används jquery för?

FredrikRissanen replied:
Den följde med automatiskt när jag importerade bootstrap. Men den behövs faktiskt inte! Tar bort den.

maansthoernvik commented on frontend/src/polyfills.ts:
```diff
@@ -1,3 +1,7 @@
+/***************************************************************************************************
+ * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
+ */
+import '@angular/localize/init';
```
Was ist das?

FredrikRissanen replied:
Bra fråga... en del av bootstrap tror jag.

maansthoernvik reviewed (commented):
Ser bra ut!

Commit:
- Fix help output when no arguments are provided to manage.py.
- Section build workflow.
- Add Angular build to workflow.

Commit:
Add django project check to build workflow.

Commit:
* Add django project check to build workflow.
* Refactor frontend service for hume to align with home handling of data.
* Consumer shall be based on HUME UUID rather than HOME ID to avoid
  expensive DB calls on each hub message received.
* Frontend websocket shall refresh group memberships every group_expiry seconds
  located in Django settings for the Redis channel layer.

Commit:
Merge branch 'cleanup' of github.com:megacorpincorporated/hint into cleanup

Commit:
Remove httpservice, use httpclient. Fix linting issues in frontend.

Commit:
Generate local.py during build to be able to run system check.

Commit:
Add authentication check to websocket consumer.

Commit:
Accept before disconnecting unauthorized WS connections to be able to check close codes.

Commit:
- EventService now returns a subscription ID for the subscribed event, rather than having to provide it.
- Input HUME UUID will be checked when websocket consumer orders a subscription. If the user does not
  own that HUME, the websocket will be closed.
- User overview now shows first and last name.
- Remodel user service according to hume and home services, returning a Promise for data-fetching
  methods.

Commit:
Add linting rule to disallow console statements, except warnings and errors.

Commit:
Align components object literals.

Commit:
- Remove bootstrap mentions in index.html, they are all provided by ngBootstrap.
- Update angular-eslint.

Commit:
Update local angular CLI version.

Commit:
Update angular core.

Commit:
Run npm update.

Commit:
Flesh out consumer views module.

FredrikRissanen reviewed (approved):
very gut very gut

Commit:
Bump rabbitmq-client

Commit:
Fix linting issue

Commit:
Handle ConsumeOK
